scripts/create_workspace.py

```diff
@@ -0,0 +1,26 @@
+# this script is called by AzureML notebooks to create workspace config file for the first run 
+subscription_id = os.getenv("SUBSCRIPTION_ID", default="<my-subscription-id>")
+resource_group = os.getenv("RESOURCE_GROUP", default="<my-resource-group>")
+workspace_name = os.getenv("WORKSPACE_NAME", default="<my-workspace-name>")
+workspace_region = os.getenv("WORKSPACE_REGION", default="eastus2")
+
+try:
+    ws = Workspace.from_config()
+except:
+    try:
+        ws = Workspace(
```

Comment:
This does not actually create the workspace, we should add a function to create the workspace if it does not exist.

Reply:
fixed